lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1807,6 +1809,23 @@ protected function getPartialIndexSQL(Index $index)
     }
 
     /**
+     * Adds SQL for including columns in a covering index.
+     *
+     * @param \Doctrine\DBAL\Schema\Index $index
+     *
+     * @return string
+     */
+    protected function getCoveringIndexWithColIncludesSQL(Index $index)
```

Comment:
With the return type hint, did you mean something like - `protected function getCoveringIndexWithColumnIncludesSQL(Index $index): bool { ... }`?

Also, was the override meant to apply to the `supportsCoveringIndexWithColumnIncludesSQL()` function?